roseau/load_flow/models/lines/parameters.py

```diff
@@ -218,10 +207,14 @@ def _sym_to_zy(
                 Phase to neutral susceptance (siemens/km)
 
         Returns:
-            The impedance and admittance matrices and the line model. The line model may be downgraded from
-            SYM_NEUTRAL to SYM if the model of the neutral is not possible.
+            The impedance and admittance matrices.
```

Comment:
Maybe keep a note about downgrading in the docstring? I know there is a note in the models page but it might be more useful to have here as well.